src/main/scala/akka/persistence/cassandra/query/scaladsl/CassandraReadJournal.scala

```diff
@@ -106,9 +106,12 @@ class CassandraReadJournal(system: ExtendedActorSystem, config: Config)
 
   @volatile private var sessionUsed = false
 
+  private val metricsCategory = s"cassandra.readjournal.${system.name}.hash.${System.identityHashCode(this)}"
```

Comment:
I think there is a bit of miscommunication. I can work on the akka issue. But for now, If I hardcode it, it will not be unique. So, What should I do now?